addons/calendar/wizard/calendar_provider_sync_wizard.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+from odoo.addons.base.models.ir_module import assert_log_admin_access
+
+
+class CalendarProviderSyncWizard(models.TransientModel):
+    _name = 'calendar.calendar.provider.sync.wizard'
+    _description = 'Configure and sync external calendar Wizard'
+
+    external_calendar_provider = fields.Selection([
+        ('google', 'Google'), ('microsoft', 'Outlook')],
+        "Choose an external calendar to configure", default='google')
+
+    # Allow to sync with eventually existing ICP keys without creating them if respective module is not installed
+    cal_google_client_id = fields.Char(
+        "Google Client_id",
+        default=lambda self: self.env['ir.config_parameter'].get_param('google_calendar_client_id'))
```

Comment:
Ideally the config param keys should not be used here.
But I don't see a way to make it easily extendable, so keep it as is for now.